prow/plugins/lifecycle/close.go

```diff
@@ -26,27 +26,26 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-const pluginName = "close"
-
 var closeRe = regexp.MustCompile(`(?mi)^/close\s*$`)
 
 func init() {
-	plugins.RegisterGenericCommentHandler(pluginName, handleGenericComment, nil)
+	plugins.RegisterGenericCommentHandler("close", deprecatedCloseHandleComment, nil)
 }
 
-type githubClient interface {
+type closeClient interface {
 	CreateComment(owner, repo string, number int, comment string) error
 	CloseIssue(owner, repo string, number int) error
 	ClosePR(owner, repo string, number int) error
 	IsMember(owner, login string) (bool, error)
 	AssignIssue(owner, repo string, number int, assignees []string) error
 }
 
-func handleGenericComment(pc plugins.PluginClient, e github.GenericCommentEvent) error {
-	return handle(pc.GitHubClient, pc.Logger, &e)
+func deprecatedCloseHandleComment(pc plugins.PluginClient, e github.GenericCommentEvent) error {
+
+	return handleClose(pc.GitHubClient, pc.Logger, &e, deprecatedWarn)
```

Comment:
I don't think that the `deprecatedWarn` variable is needed.
Since this is called from `deprecatedCloseHandleComment` we should always use a value of `true` here. When we deprecate the close and reopen plugins, instead of setting `deprecatedWarn` to `false` we can just delete the deprecated event handlers entirely.

`deprecatedWarn` is not needed in the `lifecycleHandleGenericComment` function either.

Reply:
This is for readability. `deprecatedWarn` is more readable than `false`